robot/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 import importlib
 
-from robot.sheepdog_trials.markers import BASE_MARKER
+from robot.marker_setup.markers import BASE_MARKER
```

Comment:
Am I miss understanding or is this import duplicated later on?

Reply:
I'm not sure, I've made as few changes as possible to the code to avoid breaking anything from previous years.